src/main/resources/templates/create-book.html

```diff
@@ -1,54 +1,48 @@
 <!DOCTYPE HTML>
 <html xmlns:th="http://www.thymeleaf.org" xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
-      layout:decorate="~{layout}">>
-
-<body>
-    <div layout:fragment="content">
-        <h1>Create Book</h1>
-        <form action="#" th:action="@{/addbook}" th:object="${books}" method="post">
-            <p>
-                <label for="name">Name:</label>
-                <input id="name" type="text" minlength="2" th:field="*{name}">
-                <span class="error-input" th:if="${#fields.hasErrors('name')}" th:errors="*{name}"></span>
-            </p>
-            <p>
-                <label for="description">Description:</label>
-                <textarea id="description" name="description" minlength="2" th:field="*{description}" cols="17" rows="5"></textarea>
-                <span class="error-input" th:if="${#fields.hasErrors('description')}" th:errors="*{description}"></span>
-            </p>
-            <p>
-                <label for="ISBN">ISBN:</label>
-                <input id="ISBN" type="text" minlength="13" maxlength="13" th:field="*{ISBN}">
-                <span class="error-input" th:if="${#fields.hasErrors('ISBN')}" th:errors="*{ISBN}"></span>
-            </p>
-            <p>
-                <label for="picture">Picture:</label>
-                <input id="picture" type="file" th:field="*{picture}">
-                <span class="error-input" th:if="${#fields.hasErrors('picture')}" th:errors="*{picture}"></span>
-            </p>
-            <p>
-                <label for="author">Author:</label>
-                <input id="author" type="text" minlength="2" th:field="*{author}">
-                <span class="error-input" th:if="${#fields.hasErrors('author')}" th:errors="*{author}"></span>
-            </p>
-            <p>
-                <label for="publisher">Publisher:</label>
-                <input id="publisher" type="text" minlength="2" th:field="*{publisher}">
-                <span class="error-input" th:if="${#fields.hasErrors('publisher')}" th:errors="*{publisher}"></span>
-            </p>
-            <p>
-                <label for="inventory">Inventory:</label>
-                <input id="inventory" type="number" min="0" th:field="*{inventory}">
-                <span class="error-input" th:if="${#fields.hasErrors('inventory')}" th:errors="*{inventory}"></span>
-            </p>
-            <p>
-                <label for="price">Price:</label>
-                <input id="price" type="number" step="0.01" min="0" th:field="*{price}">
-                <span class="error-input" th:if="${#fields.hasErrors('price')}" th:errors="*{price}"></span>
-            </p>
-            <p><input id="submit" type="submit" value="Create Book" /> <input type="reset" value="Reset" /></p>
-        </form>
-    </div>
-</body>
```

Comment:
these indentation changes make it hard to see what's being changed by this PR, for future PR's please leave the indentation as is and do the indentation change in a separate PR if necessary. However, I find it's pretty common anyways for the body, head, etc. to not be indented within the html element since this just adds an extra level of indentation and makes you have to deal with line wrappings even more

Reply:
html is typically not indented because it's usually generated or compressed. I indented because i find it easier to follow the code (know where things begin and end).

Reply:
yeah I'm ok with that being indented I'm just saying it made the PR more difficult to review, and here's what I was talking about for some people also having the opinion of not indenting html, body, and head since they are trivial: https://stackoverflow.com/a/29900324